File: samples_generated_code/sample_1/phi4/baseline_code_c/code_row_81.c
#include <stdio.h>

int getValueFromArray(int arr[], int size, int index) {
    if (index >= 0 && index < size) {
        return arr[index];
    }
    return -1; // Indicate error with an invalid index
}

int main() {
    int id_sequence[] = {10, 20, 30, 40, 50}; // Example array
    int size = sizeof(id_sequence) / sizeof(id_sequence[0]);
    
    printf("Enter the index: ");
    int index;
    if (scanf("%d", &index) != 1) {
        return -1; // Indicate error with input failure
    }

    int result = getValueFromArray(id_sequence, size, index);
    return result;
}

File: samples_generated_code/sample_3/phi4/baseline_code_java/code_row_81.java
import java.util.Scanner;

public class Main {
    public static int getValueFromArray(int[] arr, int size, int index) {
        if (index >= 0 && index < size) {
            return arr[index];
        }
        return -1; // Return a default value or handle error as needed
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int idSequence[] = {10, 20, 30, 40, 50};
        
        System.out.print("Enter an index: ");
        int index = scanner.nextInt();
        
        int result = getValueFromArray(idSequence, idSequence.length, index);
        System.exit(result);
    }
}